validators/os_validator_unix.go

```diff
@@ -39,11 +40,13 @@ func (o *OSValidator) Name() string {
 
 // Validate is part of the system.Validator interface.
 func (o *OSValidator) Validate(spec SysSpec) ([]error, []error) {
-	os, err := exec.Command("uname").CombinedOutput()
+	var utsname unix.Utsname
+	err := unix.Uname(&utsname)
 	if err != nil {
 		return nil, []error{fmt.Errorf("failed to get OS name: %w", err)}
 	}
-	if err = o.validateOS(strings.TrimSpace(string(os)), spec.OS); err != nil {
+	os := strings.TrimSpace(unix.ByteSliceToString(utsname.Sysname[:]))
```

Comment:
here it's possible to call getKernelRelease as well?

Reply:
`getKernelRelase` returns `utsname.Release` (i.e. the kernel version) while this needs `uts.Sysname` (i.e. the kernel name), so I don't think we can use `getKernelRelase` here.